Cargo.toml

```diff
@@ -11,6 +11,7 @@ categories = ["no-std", "rust-patterns"]
 license = "MIT OR Apache-2.0"
 
 [dev-dependencies]
+never-say-never = "6.6.666"
 
 [dependencies]
 trybuild = "1.0.110"
```

Comment:
You obviously don't need to change this here, but why did I make trybuild a regular dependency way back when? It's only used for tests 😅